### ISSUE TITLE
FormGroup: Using switch prop results in warning: Received `true` for a non-boolean attribute `switch`

### ISSUE THREAD

**Issue by mikegoatly** [reactstrap/reactstrap#2415]
- components: `name`
- reactstrap version `#9.0.1`
- import method `typescript, import { FormGroup } from 'reactstrap' - whichever that is :)`
- react version `#17.0.2`
- bootstrap version `#5.1.3`

### What is happening?

When using the `switch` prop with `FormGroup`:

```
<FormGroup switch>
...
</FormGroup>
```

I get this warning in the browser:

```
Warning: Received `true` for a non-boolean attribute `switch`.
      
      If you want to write it to the DOM, pass a string instead: switch="true" or switch={value.toString()}.
          at div
          at FormGroup (C:\dev\Portal\src\Coeo.Portal\clientapp\node_modules\reactstrap\src\FormGroup.js:25:5)
         ...
```

### What should be happening?

No warnings should be displayed.

### Steps to reproduce issue

1. Create a `FormGroup` and set the `switch` prop.
2. Run the app

### Error message in console

See above.

### Code

https://reactstrap-py5tda.stackblitz.io

(Check the console pane to see the warning)

**Comment by mikegoatly** [comment]
For visibility here, @jfhenon mentioned this workaround in the pull request:

> workaround: you can remove the switch prop from FormGroup and replace with the bootstrap classname:
> ` <FormGroup className="form-check form-switch">`

**Comment by davidacevedo** [comment]
Fix should be a part of `9.0.2`
